src/parts/display.class.ts

```diff
@@ -0,0 +1,20 @@
+import { Value } from "./value.class";
+
+export class Display {
+    constructor(private readonly element: HTMLInputElement) {}
+
+    setValue(value: Value) {
+        this.element.value = value.toString();
+    }
+
+    async blink() {
+        const value = this.element.value;
+
+        this.element.value = "";
```

Comment:
This would have a side effect if the width of the element is variable?
e.g. setting '' may change a width of the element?

If we are sure the the width doesn't change, plz put a comment.